docs/testrunner-toolkit/saucectl.md

```diff
@@ -1,188 +1,355 @@
+
+```bash title="Interactive Example"
+$ saucectl init
+12:13:20 INF Start Init Command
+? Select region: us-west-1
+? Select framework:  [Use arrows to move, type to filter]
+> cypress
+ playwright
+ puppeteer
+ testcafe
+ espresso
+ xcuitest
+```
+
+```bash title="Batch Example"
+$saucectl init -r us-west-1 -f cypress -b chrome
 ```
 
-### `run`
+`saucectl` supports the following configuration flags as inline specifications.
+
+
+### `-a, --accessKey`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+The authentication access key associated with the Sauce Labs user account making this request. If you have not set your authentication credentials as environment parameters or generated a `credentials.yml` file, this value is required.
+
+---
+
+### `-app`
+<p><small>| OPTIONAL | STRING | XCUITEST/ESPRESSO ONLY |</small></p>
+
+The path to a valid mobile application to test.
+
+---
+
+### `--artifacts.download.when`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Specifies when and under what circumstances to download artifacts. Valid values are:
+
+* `always`: Always download artifacts.
+* `never`: Never download artifacts.
+* `pass`: Download artifacts for passing suites only.
+* `fail`: Download artifacts for failed suites only. (default value)
+
+---
+
+### `-b, --browserName`
+<p><small>| REQUIRED | STRING | WEB-APP ONLY |</small></p>
+
+The name of the browser in which to run tests.
+
+---
+
+### `--cypress.config`
+<p><small>| REQUIRED | STRING | CYPRESS ONLY |</small></p>
+
+The file path to the Cypress configuration file (typically `cypress.json`).
+
+---
+
+### `--device`
+<p><small>| OPTIONAL | STRING | XCUITEST/ESPRESSO ONLY |</small></p>
 
-__Description__: Executes tests based on information in the configuration file ([`.sauce/config.yml`](/testrunner-toolkit/configuration)).
+Find a real device for this test matching the device name or portion of the name provided. Use the [Get Devices API Request](https://docs.saucelabs.com/dev/api/rdc#get-devices) for a list of supported real devices in your region.
 
-__Example__:
+---
+
+### `--emulator`
+<p><small>| OPTIONAL | STRING | ESPRESSO ONLY |</small></p>
+
+The name of the virtual device emulator for the test. Check the list of [supported virtual devices](https://app.saucelabs.com/live/web-testing/virtual) for valid names.
+
+---
+
+### `-f, --framework`
+<p><small>| REQUIRED | STRING |</small></p>
+
+The framework for which this configuration is intended.
+
+---
+
+### `-v, --frameworkVersion`
+<p><small>| REQUIRED | STRING | WEB APPS ONLY |</small></p>
+
+The version of the framework that is compatible with the tests defined in this configuration.
+
+---
+
+### `-h, --help`
+
+Usage information for the `init` command.
+
+---
+
+### `-p, --platformName`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+A specific operating system and version on which to run the specified browser and test suite. Defaults to a platform that is supported by `saucectl` for the specified browser.
```

Comment:
There is no default value to that one.
Notice that this is the place where you can specifiy `docker` as a platform.